File: R/tests/testthat/test_bart_par4_noise.R
context("Test bart_par4_noise")
library(hBayesDM)

test_that("Test bart_par4_noise", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(bart_par4_noise(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})

File: Python/tests/test_bart_par4_noise.py
import pytest

from hbayesdm.models import bart_par4_noise


def test_bart_par4_noise():
    _ = bart_par4_noise(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()
